doc/release-notes.md

```diff
@@ -102,6 +105,13 @@ a zero-fee transaction to begin confirmation within nblocks. Returns -1 if not
 enough free transactions have been observed to compute a good
 estimate.
 
+Relay policy changes
```

Comment:
Let's combine this with the `-datacarrier` changes (e.g. under `Mining enhancements`). Mining and relay policy, after all, coincides.